packages/instant/src/containers/selected_asset_buy_button.ts

```diff
@@ -7,6 +7,7 @@ import { Dispatch } from 'redux';
 import { Action, actions } from '../redux/actions';
```

Comment:
In development, the buy button logic is now in `selected_asset_buy_order_state_buttons`

Reply:
Lol git figured it out

Reply:
Nice :)